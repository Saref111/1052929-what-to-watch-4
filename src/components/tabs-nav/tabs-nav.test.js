import React from "react";
import renderer from "react-test-renderer";
import TabsNav from "./tabs-nav.jsx";

test(`Snapshot test for TabsNav`, () => {
  const tree = renderer.create(<TabsNav page={1} handleClick={() => {}} />);

  expect(tree).toMatchSnapshot();
});
