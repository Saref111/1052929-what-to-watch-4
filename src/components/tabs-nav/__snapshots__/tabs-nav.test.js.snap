// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test for TabsNav 1`] = `
<nav
  className="movie-nav movie-card__nav"
>
  <ul
    className="movie-nav__list"
  >
    <li
      className="movie-nav__item "
    >
      <a
        className="movie-nav__link"
        href=""
        id={0}
        onClick={[Function]}
      >
        Overview
      </a>
    </li>
    <li
      className="movie-nav__item movie-nav__item--active"
    >
      <a
        className="movie-nav__link"
        href=""
        id={1}
        onClick={[Function]}
      >
        Details
      </a>
    </li>
    <li
      className="movie-nav__item "
    >
      <a
        className="movie-nav__link"
        href=""
        id={2}
        onClick={[Function]}
      >
        Reviews
      </a>
    </li>
  </ul>
</nav>
`;
